Examples/Algorithms/Vertexing/src/IterativeVertexFinderAlgorithm.cpp

```diff
@@ -93,8 +93,8 @@ ActsExamples::ProcessCode ActsExamples::IterativeVertexFinderAlgorithm::execute(
   VertexSeeder::Config seederCfg(ipEst);
   VertexSeeder seeder(seederCfg);
   // Set up the actual vertex finder
-  VertexFinder::Config finderCfg(vertexFitter, linearizer, std::move(seeder),
-                                 ipEst);
+  VertexFinder::Config finderCfg(vertexFitter, std::move(linearizer),
```

Comment:
should we pass the logger down to the linearizer in the algorithm?

Reply:
yeah exactly. I think there is a logger available in the algorithm which could be passed down?

Reply:
Not really easy. You can't pass in the algorithm logger because it's a unique ptr. There's also no good way to get the log level from the logger. I can work around this if we think it's needed.

Reply:
no thats fine. I just thought it might be nice to pass the logger down